sdk/program/src/syscalls/definitions.rs

```diff
@@ -77,53 +47,3 @@ define_syscall!(fn sol_get_epoch_rewards_sysvar(addr: *mut u8) -> u64);
 
 // this cannot go through sol_get_sysvar but can be removed once no longer in use
 define_syscall!(fn sol_get_fees_sysvar(addr: *mut u8) -> u64);
-
-#[cfg(target_feature = "static-syscalls")]
-pub const fn sys_hash(name: &str) -> usize {
-    murmur3_32(name.as_bytes(), 0) as usize
-}
```

Comment:
Although I don't see any other users in the repo, this is a breaking change -- could you add a re-export for `sys_hash` in this file?